res/autoload.template

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Would this file be installed on systems via RPM of this pkg and be used by other RPM spec files like ` phpab -t /install_location/autoload.template`?